_includes/manuals/1.18/3.1.3_puppet_versions.md

```diff
@@ -1,4 +1,5 @@
 Foreman integrates with Puppet and Facter in a few places, but generally using a recent, stable version will be fine.  The exact versions of Puppet, Puppet Server and Facter that Foreman is compatible with are listed below.
+Foreman 1.18 is the last version that will support Puppet < 4.0.
```

Comment:
👎 
Can't we make this `< 3.8`?

Reply:
Even Puppet 4 will be end of life later this year. Can we stick to supporting just one out-of-date version? :P

Reply:
Given our installer requires puppet 4 since a few versions, I'm inclined to say we no longer fully support `< 4.0`. We have partial support for older versions for now.

Reply:
Can we clarify the doc then? If we're leaving in the report importers etc then we should probably say that the functionality will remain but is unsupported / untested on < 3.8